### DISCUSSION TITLE
[i18n] [icon] Suggest adding a "global" or "translation" icon

### DISCUSSION THREAD

**Discussion by z621537** [element-plus/element-plus#16951] [Feature Request]
Feature type: **`i18n`**

## Details

- Enhanced component name: `icon`

## Description

I would like to have a icon "Global" or "Translation", I'm doing International project now. But I don't find a suitable icon. Thank you.

### Additional comments

_(empty)_

**Comment by xxholly32** [comment]
try https://icones.js.org/

**Reply by z621537** [reply]
this is really a big hub
